includes/class-sensei-grading.php

```diff
@@ -120,12 +120,17 @@ public function load_data_table_files() {
 	 * load_data_object creates new instance of class
 	 *
 	 * @since  1.3.0
+	 * @deprecated 3.3.0  Use constructors instead.
+	 *
 	 * @param  string    $name          Name of class
 	 * @param  integer   $data          constructor arguments
 	 * @param  undefined $optional_data optional constructor arguments
 	 * @return object                 class instance object
 	 */
 	public function load_data_object( $name = '', $data = 0, $optional_data = null ) {
+		// User constructors directly.
+		_deprecated_function( __METHOD__, '2.0.0' );
```

Comment:
Version would be `3.3.0`. Could do `_deprecated_function( __METHOD__, '3.3.0', "new Sensei_Grading_{$name}" );` to add the replacement function.

Reply:
Fixed in c73913a55f49492a63b34b1c7c25b4c5bf57d3ce